commons/src/main/scala/com/lynbrookrobotics/potassium/commons/drivetrain/Drive.scala

```diff
@@ -9,10 +9,18 @@ trait Drive {
   type DriveSignal
   type DriveVelocity
 
+  type DrivetrainHardware
```

Comment:
We want to take hardware and props as parameters in all controllers/tasks, for unit testing purposes, so here we just declare an abstract type for what type of hardware and props the methods should be receiving.

Now that I think of it, we can probably just eliminate these types and take the unicycle/twosided hardware and props directly, and this should work since the final hardware/props case class will extend those types.